routers/web/repo/view.go

```diff
@@ -698,17 +698,11 @@ func checkCitationFile(ctx *context.Context, entry *git.TreeEntry) {
 
 // Home render repository home page
 func Home(ctx *context.Context) {
-	if setting.EnableFeed {
-		isFeed, _, showFeedType := feed.GetFeedType(ctx.Params(":reponame"), ctx.Req)
-		if isFeed {
-			feed.ShowRepoFeed(ctx, ctx.Repo.Repository, showFeedType)
-			return
-		}
-
-		ctx.Data["EnableFeed"] = true
-		ctx.Data["FeedURL"] = ctx.Repo.Repository.Link()
+	if feed.RenderRepoFeed(ctx) {
```

Comment:
dont return a bool just use:
```
if ctx.Written() {
		return
	}
	```

Reply:
this also allow [to inject vars](https://github.com/go-gitea/gitea/pull/22719/files#r1155353519)

Reply:
feed.RenderRepoFeed no longer returns a boolean and checks ctx.Written as suggested